astroquery/esa/xmm_newton/core.py

```diff
@@ -266,5 +267,80 @@ def get_columns(self, table_name, *, only_names=True, verbose=False):
         else:
             return columns
 
+    def get_epic_metadata(self, *, target_name=None,
+                          coordinates=None, radius=None):
+        """Downloads the European Photon Imaging Camera (EPIC)
+        metadata from a given target
+
+        Parameters
+        ----------
+        target_name : string, optional, default None
+            The name of the target
+        coordinates : astropy.coordinates.SkyCoord, optinal, default None
+            The coordinates of the target in a SkyCoord object
+        radius : float, optional, default None
+            The radius to query the target in degrees
+        Returns
+        -------
+        Four astropy.table.table.Table containing the metadata
+        of the given target
+        """
+        if not target_name and not coordinates:
+                raise Exception(("Input parameters needed, "
+                                 "please provide the name "
+                                 "or the coordinates of the target"))
+
+        epic_source = {"table": "xsa.v_epic_source",
+                       "column": "epic_source_equatorial_spoint"}
+        cat_4xmm = {"table": "xsa.v_epic_source_cat",
+                    "column": "epic_source_cat_equatorial_spoint"}
+        stack_4xmm = {"table": "xsa.v_epic_xmm_stack_cat",
+                      "column": "epic_stack_cat_equatorial_spoint"}
+        slew_source = {"table": "xsa.v_slew_source_cat",
+                       "column": "slew_source_cat_equatorial_spoint"}
+
+        cols = "*"
+
+        c = coordinates
+        if not coordinates:
+            c = SkyCoord.from_name(target_name, parse=True)
+
+        if type(c) is not SkyCoord:
+            raise Exception(("The coordinates must be an "
+                             "astroquery.coordinates.SkyCoord object"))
+        if not radius:
+            radius = 0.1
+
+        query_fmt = ("select %s from %s "
+                     "where 1=contains(%s, circle('ICRS', %f, %f, %f));")
```

Comment:
please use `.format` rather than the old style, or if possible fstrings (given that we dropped the python2 support)